Makefile

```diff
@@ -6,11 +6,17 @@ PYTHON = python
 PIP = $(PYTHON) -m pip -v
 LS_R = ls -Ra1
 
-# We add ulimit -s 1024 in this Makefile to work around a very strange
-# OS X bug manifesting itself with Python 3 and old versions of GNU make.
-# This was discovered at https://github.com/sagemath/cysignals/issues/71
-# but I don't know a good analysis nor solution.
-DOCTEST = ulimit -s 1024 && $(PYTHON) -B rundoctests.py
+ifeq ($(OS),Windows_NT)
```

Comment:
I just pushed a commit to master which hopefully makes these changes to `Makefile` unneeded.

Reply:
It works for removing LN_SITE_PACKAGE. But `ulimit` don't exist on windows. I will push the rebase after some others tests

Reply:
> But ulimit don't exist on windows.

OK, so you get an error but the tests continue, right?

Reply:
> I will push the rebase after some others tests

Maybe squash your commits. It's not really helpful to have a lot of commits, especially when some just revert earlier commits.

Reply:
> OK, so you get an error but the tests continue, right?

Yes ulimit don't look to be blocking, but i have another error crashing distcheck with my current rebase.

Reply:
Maybe instead of doing these kind of hacks, we could just say that one should run `make distcheck` on Windows. I think that it's fine if we run `make distcheck` on at least one platform, but not all.